cigaradvisor/blocks/hero/hero.js

```diff
@@ -14,6 +14,7 @@ function buildSearch() {
 }
 
 export default async function decorate(block) {
+  block.classList.add('under-nav');
```

Comment:
This option is an Author decision, it's not something that can be assumed for every hero.

Reply:
I think the hero variation that doesn't go under the nav is on article pages. We are handling that separately as part of article header.

Reply:
Maybe, but you can't assume that every hero is under the nav.

Reply:
fixed